wet-boew/eqht-css/index.json-ld

```diff
@@ -0,0 +1,46 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+		"title": {
+		"en": "Equal height (pure CSS)",
+		"fr": "Égalisation des hauteurs (pure CSS)"
+	},
+	"description": {
+		"en": "Equalises the height of elements that are on the same baseline",
+		"fr": "Égalise la hauteur d'éléments étant sur la même ligne de base"
+	},
+	"modified": "2024-05-17",
+	"componentName": "eqht-css",
+	"status": "stable",
+	"pages": {
+		"examples": [
+			{
+				"title": "Equal height (pure CSS)",
+				"language": "en",
+				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/eqht-css/eqht-css-en.html"
+			},
+			{
+				"title": "Égalisation des hauteurs (pure CSS)",
+				"language": "fr",
+				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/eqht-css/eqht-css-fr.html"
+			}
+		],
+		"docs": [
+			{
+				"title": "Equal height (pure CSS)",
+				"language": "en",
+				"url": "https://wet-boew.github.io/wet-boew/docs/ref/eqht-css/eqht-css-en.html"
+			},
+			{
+				"title": "Égalisation des hauteurs (pure CSS)",
+				"language": "fr",
+				"url": "https://wet-boew.github.io/wet-boew/docs/ref/eqht-css/eqht-css-fr.html"
+			}
+		]
```

Comment:
Let's use the docs of the other equal height and add those link under "other" group.

```suggestion
		"other": [
			{
				"title": "Documentation - Equal height (pure CSS)",
				"language": "en",
				"url": "https://wet-boew.github.io/wet-boew/docs/ref/eqht-css/eqht-css-en.html"
			},
			{
				"title": "Documentation - Égalisation des hauteurs (pure CSS)",
				"language": "fr",
				"url": "https://wet-boew.github.io/wet-boew/docs/ref/eqht-css/eqht-css-fr.html"
			}
		]
```

Reply:
@duboisp Why not have the documentation for the CSS Equalheight and JS Equalheight together? I don't understand why they have to by in a separate section.